tests/snapshots/assets__check_file@cetz-tree.typ-40.snap

```diff
@@ -107,10 +109,12 @@ snapshot_kind: text
       } else if (
         type(content) != typst-content
       ) {
-        panic("Unsupported content type "
-          + type(content)
-          + "! "
-          + "Provide your own `draw-node` implementation.")
+        panic(
+          "Unsupported content type "
+            + type(content)
+            + "! "
+            + "Provide your own `draw-node` implementation.",
+        )
```

Comment:
this looks nice. But I guess the trailing comma here is unneeded? I'm not quite sure at this moment

Reply:
needless, but the args are multiline, thus a trailing comma is added